app/models/section.rb

```diff
@@ -13,7 +13,7 @@ class Section < ApplicationRecord
         foreign_key: :section_id,
         association_foreign_key: :connection_id
 
-  validates :name, uniqueness: { case_sensitive: false }
```

Comment:
Nope. We can validate name uniqueness on the sectionhome node but I wasn't sure if we wanted that or not.